### DISCUSSION TITLE
Error saving cipher

### DISCUSSION THREAD

**Discussion by suricloud** [dani-garcia/vaultwarden#1511] [Q&A]
Hi Just installed it on my vps with ubuntu 20.04 following this guide ( https://www.linode.com/docs/guides/how-to-self-host-the-bitwarden-rs-password-manager/) 

Installed succesdully but cant save any passwords, i do get the erro of "  Error saving cipher "
How to fix this?

### Deployment environment

* bitwarden_rs version:

* Install method:

* Clients used: 

* Reverse proxy and version: 

* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

**Comment by BlackDex** [comment]
For that we need more information that you are giving here. It clearly states what we need when you are creating this ticket.
So please provide logs from the reverse proxy if possible, from bitwarden.log, else we can't help at all.

**Reply by suricloud** [reply]
what is the directory to see the bitwarden logs?
